tests/fips/openssl/openssl_pubkey_dsa.pm

```diff
@@ -50,11 +56,13 @@ sub run {
         # A source of random numbers is required for DSA, so get message digest first
         assert_script_run "openssl dgst -$dgst_alg $file_raw | awk '{print $2}' > $file_dgt";
 
-        # Sign message digest with private key
-        assert_script_run "openssl dgst -dss1 -sign $dsa_prikey $file_dgt > $file_sig";
+        # Sign message digest with private key and verify signature with public key
+        # openssl >= 1.1.x (sha256)
+        # openssl >= 1.0.x (dss1)       
+        my $algo = ($openssl_version eq "1.1") ? "sha256" : "dss1";
```

Comment:
What if there will be openSSL version 1.2.x in future?
```
        my $algo = ($openssl_version ge "1.1") ? "sha256" : "dss1";
```

Reply:
Base on the openssl wiki, dss1 is deprecated from 1.1.0 and it is no longer support from 1.1.x. 
https://en.wikipedia.org/wiki/OpenSSL
Will it be possible to change the "eq" can be replaced to ">=" ?

Reply:
`ge` is `>=`

Reply:
Done the modification. Thanks.